### ISSUE TITLE
Consider inclusion of ICDO3

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-disease-ontology#156]
http://codes.iarc.fr/

**Comment by cmungall** [comment]
Do mappings already exist from NCIT?

**Comment by mellybelly** [comment]
yes there seem to be some, but not sure how complete. There is a class and a property for ICD-O-3:
http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#ICD-O-3_Code_Property
no uses
http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#ICD-O-3_Code
1224 uses
